arangod/Graph/Providers/SingleServerProvider.cpp

```diff
@@ -88,10 +88,10 @@ SingleServerProvider::SingleServerProvider(arangodb::aql::QueryContext& queryCon
                                            BaseProviderOptions opts,
                                            arangodb::ResourceMonitor& resourceMonitor)
     : _trx(std::make_unique<arangodb::transaction::Methods>(queryContext.newTrxContext())),
-      _stats{},
+      _opts(std::move(opts)),
       _cache(_trx.get(), &queryContext, resourceMonitor, _stats,
              opts.collectionToShardMap()),
```

Comment:
This needs to be _opts now. The opts has just been moved the line before.
```suggestion
             _opts.collectionToShardMap()),
```